site/app/models/gradeable/LateDayInfo.php

```diff
@@ -282,6 +289,14 @@ public function getLateDayException() {
         return $this->late_day_exceptions;
     }
 
+    /**
+     * Gets the number of late days the student gets extra for this gradeable
+     * @return string
+     */
+    public function getReasonForException() {
```

Comment:
Since there's just one type here, it would be good to change this to a native PHP type declaration

Reply:
Could you explain this too?